lib/benchee/formatters/console.ex

```diff
@@ -70,7 +70,17 @@ defmodule Benchee.Formatters.Console do
       |> Map.merge(options)
 
     scenarios
-    |> Enum.group_by(fn scenario -> scenario.input_name end)
+    |> Enum.reduce([], fn scenario, grouped ->
+      case List.keyfind(grouped, scenario.input_name, 0) do
+        {_, group} ->
+          new_tuple = {scenario.input_name, [scenario | group]}
+          List.keyreplace(grouped, scenario.input_name, 0, new_tuple)
+
+        _ ->
+          [{scenario.input_name, [scenario]} | grouped]
```

Comment:
again a bit long for my taste to be inside a pipe, I'd take it and put it into some function which forces us to name it which could be helpful :)